### ISSUE TITLE
Trigger Window to show last trigger check and result

### ISSUE THREAD

**Issue by khandelwalpiyush** [lithnet/miis-autosync#101]
Nice to have if the triggers window can show when the last time the trigger check / was run and what run profile it caused to run if it did so..

**Comment by ryannewington** [comment]
Yes good idea. Ive been thinking about how to do this but haven't come up with any good ideas on how to render it yet

**Comment by khandelwalpiyush** [comment]
Extension to the current trigger type and description with last run and result
